generate.go

```diff
@@ -66,6 +67,11 @@ func makeFormatInfo(cmd *cobra.Command, opts Options) formatInfo {
 	fmtInfo.Groups = makeGroupsInfo(cmd)
 
 	fmtInfo.GlobalFlagsBlock = makeFlagsBlock(cmd.PersistentFlags())
+	if fmtInfo.GlobalFlagsBlock == "" {
+		fmtInfo.HasGlobalFlags = false
+	} else {
+		fmtInfo.HasGlobalFlags = true
+	}
```

Comment:
This can be simplified to:

```
fmtInfo.HasGlobalFlags = fmtInfo.GlobalFlagsBlock != ""
```

But it seems that HasGlobalFlags is not used anyway and we should just remove it.

Reply:
I did not realize that I never used that property in the template - still learning how those work. Removed.